portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,10 @@
  */
 function addRandomGreeting() {
```

Comment:
I'd change this function name to `addRandomFood` because the action has changed.

Reply:
And I would change the comment above describing the function's actions.